README.md

```diff
@@ -47,6 +47,13 @@ try:
 
         docker compose up --build
 
+### Errors starting the app and gulp containers 
+
+In RedHat based systems e.g. Red Hat Enterprise Linux, Fedora Core, etc you may need to set SELinux to Permissive for the Entrypoint script and other bash scriptsto have the correct permissions. Try: 
+
+        sudo getenforce
+        sudo setenforce permissive
```

Comment:
Looks like we've got some inconsistent styling in this file but could you outdent this to the first column and quote it as a command block please?

Thanks for the contribution!